BatteryMonitoring/CurrentMeasurement.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace BatteryMonitoring
+{
+    public class CurrentMeasurement
+    {
+
+        readonly Dictionary<string, int> _currentRanges = new Dictionary<string, int>();
+
+        public Dictionary<string, int> GetRangesWithReadings(List<int> currentValues)
+        {
+            if (currentValues == null)
+                throw new ArgumentNullException("Measurement cannot be null");
+
+            if (currentValues.Count == 0)
```

Comment:
what is the complexity of GetRangesWithReadings() function
seperate null heck

GetRangesWithReadings() implementation incomplete

Reply:
Hi sir 
Implementation not done in latest commit.